compiler/qsc_rca/tests/literals.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
```

Comment:
This file doesn't have any tests for string literals or string interpolation. Literals might fit here, but I wonder if there needs to be a string.rs file that handles both, since strings can start to get complex in their own right with nested interpolation, concatenation, equality comparisons, etc.

Reply:
I'll create a string.rs test file and add tests with interpolation, concatenation and equality comparisons.

Reply:
Created strings.rs specifically to tests RCA on strings and consolidated the tests for literals in other data types into the types.rs file.